menu/translations/lxqt-settings-system_si.directory.yaml

```diff
@@ -0,0 +1 @@
+Desktop Entry/Name: ""
```

Comment:
Translation lost during refactor?

Reply:
Was my thought too, but he created the file but didn't translate it yet:
https://translate.lxqt-project.org/translate/lxqt-configuration/system-settings-in-lxqt-configuration-center/si/?q=state:%3Ctranslated